backend/src/main/kotlin/hu/bme/sch/cmsch/component/token/TokenComponent.kt

```diff
@@ -123,4 +128,28 @@ class TokenComponent(
         fieldName = "Alapértelmezett test ikon", description = "Azoknak a teszt tokeneknek ahova nincs egyedi megadva (0-100 egész szám)"
     )
 
+    /// -------------------------------------------------------------------------------------------------------------------
+
+    val reportGroup = SettingProxy(componentSettingService, component,
+        "styleGroup", "", type = SettingType.COMPONENT_GROUP, persist = false, serverSideOnly = true,
+        fieldName = "Jelenléti ív",
+        description = "Jelenléti ív beállítások"
+    )
+
+    final val reportTitle = SettingProxy(componentSettingService, component,
+        "reportTitle", "GÓLYAKÖRTE 2025",
+        fieldName = "Jelenléti ív címe", description = "Ez lesz a jelenléti ív címe"
+    )
+
+    final val reportLogo = SettingProxy(componentSettingService, component,
+        "reportLogo", "/cdn/manifest/report-logo.png",
+        fieldName = "Jelenléti ív logója", description = "Ez lesz a jelenléti íven megjelenített logó (csak PNG és JPG jó)",
+        type = SettingType.IMAGE,
```

Comment:
Do we have that setting type? I didn't remember that! :D

Reply:
Yes, it's for setting manifest images, and thats why I set it like /cdn/manifest/...